src/engine/world/EngineScene.ts

```diff
@@ -1,17 +1,31 @@
-import { PerspectiveCamera, Scene } from 'three'
+import { Pass, RenderPass } from 'postprocessing'
+import { PerspectiveCamera, Scene, WebGLRenderer } from 'three'
 
 export class EngineScene {
   public scene: Scene
   public camera: PerspectiveCamera
+  public passes: Pass[]
```

Comment:
Feel like passes belong on the Scene, will be nice for A/B mixing. 
Global passes can come via a global EngineScene rather than checking for cases, separate tracking, etc. from og hedron

Reply:
sounds good!